scripts/install-crowdsec_openresty_bouncer

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+
+BLUE='\E[1;34m'
+CYAN='\E[1;36m'
+YELLOW='\E[1;33m'
+GREEN='\E[1;32m'
+RESET='\E[0m'
+
+echo -e "${BLUE}❯ ${CYAN}Installing Crowdsec OpenResty Bouncer ${YELLOW}${CROWDSEC_OPENRESTY_BOUNCER_VERSION}...${RESET}"
+
+cd /tmp
+wget "https://github.com/crowdsecurity/cs-openresty-bouncer/releases/download/v${CROWDSEC_OPENRESTY_BOUNCER_VERSION}/crowdsec-openresty-bouncer.tgz"
+mkdir -p /tmp/crowdsec
+tar xzf crowdsec-openresty-bouncer.tgz --strip 1 -C /tmp/crowdsec
+cd /tmp/crowdsec
+
+bash ./install.sh --NGINX_CONF_DIR=/etc/nginx/conf.d --LIB_PATH=/etc/nginx/lualib --CONFIG_PATH=/defaults/crowdsec --DATA_PATH=/defaults/crowdsec --docker
+sed -i 's|ENABLED=.*|ENABLED=false|' /defaults/crowdsec/crowdsec-openresty-bouncer.conf
+
```

Comment:
Do you still need the `/tmp/crowdsec` folder here? can it be removed to decrease the final image size?

Reply:
it can be removed and has been removed in the next commit. I also removed gettext which was added for the install script above.

Reply:
The Grok node uses an ifstatement so only one will be returned as well (%{EMAILADDRESS:email}|%{USERNAME:username})